Tasks/AzurePowerShellV5/AzurePowerShell.ps1

```diff
@@ -192,8 +192,9 @@ finally {
     if ($__vstsAzPSInlineScriptPath -and (Test-Path -LiteralPath $__vstsAzPSInlineScriptPath) ) {
         Remove-Item -LiteralPath $__vstsAzPSInlineScriptPath -ErrorAction 'SilentlyContinue'
     }
-
-    Import-Module $PSScriptRoot\ps_modules\VstsAzureHelpers_
+    . "$PSScriptRoot\Utility.ps1"
+    Import-Module "$PSScriptRoot\ps_modules\VstsAzureHelpers_"
     Remove-EndpointSecrets
-    Disconnect-AzureAndClearContext -ErrorAction SilentlyContinue
-}
\ No newline at end of file
+    Update-PSModulePathForHostedAgent
+    Disconnect-AzureAndClearContext -restrictContext 'True' -ErrorAction SilentlyContinue
```

Comment:
Would it be possible to reuse RemoveAzContext.ps1 script here?

Reply:
@onetocny I think we cannot reuse RemoveAzContext.ps1 as in this file we are calling Update-PSModulePathForHostedAgentLinux function but we have to call this Update-PSModulePathForHostedAgent function.